spec/controllers/equipment_models_controller_spec.rb

```diff
@@ -1,421 +1,371 @@
 require 'spec_helper'
 
-shared_examples_for 'GET show success' do
-  it { is_expected.to respond_with(:success) }
-  it { is_expected.to render_template(:show) }
-  it { is_expected.not_to set_flash }
-  it 'should set to correct equipment model' do
-    expect(assigns(:equipment_model)).to eq(model)
-  end
-  it 'should set @associated_equipment_models' do
-    mod1 = FactoryGirl.create(:equipment_model)
-    model.associated_equipment_models = [mod1]
-    get :show, id: model
-    expect(assigns(:associated_equipment_models).size).to eq(1)
-    expect(assigns(:associated_equipment_models)).to eq([] << mod1)
+describe EquipmentModelsController, type: :controller do
+  before(:each) { mock_app_config }
+  def mock_eq_model(**attrs)
```

Comment:
should think about:
* moving these methods to a helper
* having the generalized method accept traits that cause `findable` and `with_category` to run
* the fact that this feels a bit like re-implementing factory girl (I'm pretty sure that this is the better way to do it)